models/DataObject/ClassDefinition.php

```diff
@@ -509,7 +509,7 @@ public function generateClassFiles($generateDefinitionFile = true)
                             $this->getName()
                         ).'\Listing|\\Pimcore\\Model\\DataObject\\'.ucfirst(
                             $this->getName()
-                        ).' getBy'.ucfirst(
+                        ).'|null getBy'.ucfirst(
```

Comment:
Migration to regenerate class definitions?

Reply:
🏓 @NiklasBr

Reply:
@dvesh3 I added a migration